src/main/kotlin/org/stellar/walletsdk/utils/buildFeeBumpTransaction.kt

```diff
@@ -0,0 +1,23 @@
+package org.stellar.walletsdk.utils
+
+import org.stellar.sdk.FeeBumpTransaction
+import org.stellar.sdk.Server
+import org.stellar.sdk.Transaction
+
+fun buildFeeBumpTransaction(
+  feeAccount: String,
+  innerTransaction: Transaction,
+  maxFeeInStroops: Long,
```

Comment:
This is not a great name because its not the maximum fee, its the maximum _base_ fee. The transaction could be `# of ops` * `maxFeeInStroops`.

Reply:
:+1: Does `maxBaseFeeInStroops` sound good?